truffle-config.js

```diff
@@ -41,6 +42,12 @@ module.exports = {
       port: 8535,
       gas: 6.9e6,
     },
+    mainnet: {
+      network_id: 1,
+      provider: mainnetProvider,
+      gas: 7.3e6, // Lower than network limit and higher than required gas
+      gasPrice: 19000000000, // 16 gwei -> May 21st 2018, 2 * safe low
```

Comment:
**19** gwei

Reply:
This was the most expensive deployment of my life. 16 was too little lol